lib/message_express/bus/rd_kafka.rb

```diff
@@ -0,0 +1,37 @@
+require 'message_express/bus/abstract_bus'
+require 'message_express/message'
+
+module MessageExpress
+  module Bus
+    class RdKafka < AbstractBus
+      attr_reader :kafka
+
+      def initialize(kafka_client, topic: nil)
+        @kafka = kafka_client
+        @topic = topic || 'messages'
+      end
+
+      def publish(message)
+        producer_params = {
+          topic: @topic,
+          payload: message,
```

Comment:
Do not forget our messages have the following structure (I still don't fully understand why though)
```ruby
message = {
  payload: payload,
  name: 'name of the event'
}
```
so in your case 
```
  payload: message.payload
```

Reply:
ok thanks!